torch/testing/_internal/common_modules.py

```diff
@@ -350,6 +351,27 @@ def no_batch_dim_reference_mha(m, p, *args, **kwargs):
         return (output[0].squeeze(batch_dim), output[1].squeeze(0))
 
 
+def no_batch_dim_reference_rnn_gru(m, p, *args, **kwargs):
+    """Reference function for RNN supporting no batch dimensions.
+    The module is passed the input and target in batched form with a single item.
+    The output is squeezed to compare with the no-batch input.
```

Comment:
Is this description correct? IIUC the input and target are passed to this function in unbatched form and unsqueezed to create a single item batch, then they are run through the module, then the output is squeezed to compare with the no-batch *output*

Reply:
Have updated it (and the others which had same comment). Maybe I haven't phrased it nicely. PTAL :)